code/datums/gamemode/role/changeling.dm

```diff
@@ -61,7 +61,7 @@
 	to_chat(antag.current, "<img src='data:image/png;base64,[icon2base64(logo)]' style='position: relative; top: 10;'/> <span class='danger'>You are a Changeling.</span>")
 	to_chat(antag.current, "<span class='danger'>Use say \":g message\" to communicate with your fellow changelings. Remember: you get all of their absorbed DNA if you absorb them.</span>")
 	to_chat(antag.current, "<span class='info'><a HREF='?src=\ref[antag.current];getwiki=[wikiroute]'>(Wiki Guide)</a></span>")
-	if (antag.current.mind && antag.current.mind.assigned_role == "Clown")
+	if (antag.current.mind && (antag.current.mind.assigned_role == "Clown" || antag.current.mind.assigned_role == "Jester"))
```

Comment:
As far as I know, `assigned_role` will always be the job title, not the alt title. So it would be Clown, even if the crew manifest says Jester. Did you test this?

Reply:
Well, in the case of Jester it doesn't work. It has been tested and doesn't work when this is omitted, and works when included.

Reply:
you're right, I have no idea why I was clumsy one round. Does .Remove(M_CLUMSY) just inactivate the gene? Whenever you transform, your genes get activated